TaylorFramework/Modules/scissors/ScissorsExtensions.swift

```diff
@@ -352,7 +95,7 @@ Returns new string consisting of rhs copies of lhs concatenated.
 
 - parameter rhs: number of times of *lhs* to be concatenated.
 */
-func *(lhs: String, rhs: Int) -> String {
+func * (lhs: String, rhs: Int) -> String {
```

Comment:
On [Advanced Operators](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/AdvancedOperators.html#//apple_ref/doc/uid/TP40014097-CH27-ID42) page all the operator declaration have spaces. In Foundation headers however I see no spaces. Yeah, it should probably be removed